components/TIcon.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <div :class="classes" v-html="require(`../static/svg/${name}.svg`)" />
```

Comment:
I would use hero-icons

Reply:
İf use with svg name I get 500 Unexpected token '<' error. So will I change it if I configure heroicons ?